worker/src/RTC/RtpPacket.cpp

```diff
@@ -15,12 +15,12 @@ namespace RTC
 
 	/* Class methods. */
 
-	RtpPacket::SharedPtr RtpPacket::Parse(const uint8_t* data, size_t len)
+	RtpPacket* RtpPacket::Parse(const uint8_t* data, size_t len)
```

Comment:
Hm... maybe it would be more ergonomic to return UniquePtr from here? It is not clonable and will allow to avoid public `ReturnToPool` method on `RtpPacket`.

Reply:
I would honestly leave that for another PR. We have other kinds of packets (STUN, DTLS, RTCP) where we could could apply this.

Reply:
I'll do this once #8 is merged in order to reuse the allocator changes.

We can commit this before or after that.